README.md

```diff
@@ -143,6 +173,21 @@ gnfd-cmd -c config.toml head-obj gnfd://bucket-name/object-name
 // head Group
 gnfd-cmd -c config.toml head-group gnfd://groupname
 ```
+
+#### Policy Operations
+```
+// The object policy actions can be create, delete, copy, get or execute
+// It can be set with combined string like "create,delete" by --actions
+// The object policy effect can set to be allow or deny by --effect
+
+// Put object policy to group:
+gnfd-cmd -c config.toml put-obj-policy --groupId 128  --actions get,delete  gnfd://group-name/object-name
+
+// put object policy to account 
```

Comment:
```suggestion
//  Grant "put object" permission to an account
```

Reply:
updated, using "grant object operation permissions to an account"